### PR TITLE
[P-345] [fix] Embeds show pinned question on initial load

### PR DESCRIPTION
**Pull request by markflorkowski**
Fixes the issue where embeds do not show questions that were pinned before they are loaded. 

Will require updating the prod DB to add the new enum value

### CODE REVIEW

File: src/pages/embed/[uid].tsx [hunk 1]
```diff
@@ -32,25 +42,39 @@ const BrowserEmbedViewCore: React.FC<{ userId: string }> = ({ userId }) => {
...
+) => {
+  const uid = context.query.uid;
 
-  return <LazyEmbedView userId={query.uid} />;
-};
+  if (typeof uid !== "string") return { props: { success: false } };
 
-export default BrowserEmbedQuestionView;
+  const pinnedQuestion = await prisma.question
+    .findFirst({
```
Comment (hwkr) [inline src/pages/embed/[uid].tsx:69]: I feel like this call but not be very efficient if we get a lot of questions in the DB (which is likely)

**Review by hwkr — APPROVED** [review]

File: src/pages/embed/[uid].tsx (hunk 1 above)
Comment (markflorkowski) [reply src/pages/embed/[uid].tsx:69]: Resulting in a slight delay in page load time, vs the current.... not getting anything until the user pins a new question.

I think this is a non-issue. Not to mention that SQL is kind of really fast. It's not like we're working with millions of rows here.

File: src/pages/embed/[uid].tsx [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 
 import React, { useState } from "react";
 import { PusherProvider, useSubscribeToEvent } from "../../utils/pusher";
+import { prisma } from "../../server/db/client";
 
-const useLatestPusherMessage = (userId: string) => {
-  const [latestMessage, setLatestMessage] = useState<string | null>(null);
+type ServerSideProps = InferGetServerSidePropsType<typeof getServerSideProps>;
+
+const useLatestPusherMessage = (pinnedQuestion: string | null) => {
```
Comment (t3dotgg) [inline src/pages/embed/[uid].tsx]: 1. was userId string not used?
2. This should have a better name, maybe `initialPinnedQuestion` or `defaultPinnedQuestion`?

Comment (markflorkowski) [reply src/pages/embed/[uid].tsx]: 1. Correct. 
2. Good point, I'll update that.

**Comment by markflorkowski** [comment]
https://app.planetscale.com/t3/ping-ask/deploy-requests/1

**Pull request merged**